frontend/WikiContrib-Frontend/src/components/activity.js

```diff
@@ -7,6 +7,13 @@ import phabricatorPlatformIcon from '../img/phabricatorPlatformIcon.png';
 /**
  * Show all the user Commits on a specific day
  */
+window.history.pushState({page: 1}, "", "");
```

Comment:
As per the pushState's syntax [here](https://developer.mozilla.org/en-US/docs/Web/API/History/pushState), I am wondering if this line should look like this `window.history.pushState({page: 1}, "WikiContrib", "/");` and include the title and url as well?